app/models/revision.rb

```diff
@@ -36,6 +36,12 @@ class Revision < ApplicationRecord
                           association_foreign_key: "image_revision_id",
                           join_table: "revisions_image_revisions"
 
+  has_and_belongs_to_many :file_attachment_revisions,
```

Comment:
The use of this join table feels like a decision we should document at some point (maybe a section in dev doc for things to be part of an ADR)

It seems pragmatic now that we store an association of file_attachments to revisions, but this doesn't feel long term friendly to a world where we may also have html_attachments or if we need to do anything like ordering (as per publications) so probably flagged as something that is fine for now but we can anticipate issues.

Reply:
This has been documented here https://docs.google.com/document/d/1ZeWCxQ71NU_8QvVT3Y5ukl5Xei-U3DLWsc7-e5LKPB4/edit# so will resolve.